sources/postgres/README.md

```diff
@@ -1,83 +1,92 @@
-# HarbourBridge: Turnkey PostgreSQL-to-Spanner Evaluation
+# HarbourBridge: PostgreSQL-to-Spanner Evaluation and Migration
 
 HarbourBridge is a stand-alone open source tool for Cloud Spanner evaluation,
 using data from an existing PostgreSQL or MySQL database. This README provides
 details of the tool's PostgreSQL capabilities. For general HarbourBridge information
-see this [README](https://github.com/cloudspannerecosystem/harbourbridge#harbourbridge-turnkey-spanner-evaluation).
+see this [README](https://github.com/cloudspannerecosystem/harbourbridge#harbourbridge-spanner-evaluation-and-migration).
 
 ## Example PostgreSQL Usage
 
-The following examples assume `harbourbridge` alias has been setup as
-following.
-
-```sh
-git clone https://github.com/cloudspannerecosystem/harbourbridge
-cd harbourbridge
-alias harbourbridge="go run github.com/cloudspannerecosystem/harbourbridge"
-```
-
 HarbourBridge can either be used with pg_dump or it can be run directly
 on a PostgreSQL database (via go's database/sql package).
 
 ### Using HarbourBridge with pg_dump
 
-To use HarbourBridge on a PostgreSQL database called mydb using pg_dump output,run:
+The tool can used to migrate schema from an existing pg_dump file:
 
 ```sh
-pg_dump mydb | harbourbridge -driver=pg_dump
+harbourbridge schema -source=postgresql < my_pg_dump_file
 ```
 
-The tool can also be applied to an existing pg_dump file:
+You can use either of `postgresql`, `postgres`, or `pg` to `-source` to specify
```

Comment:
Suggest rephrasing:

You can use any of `postgresql`, `postgres`, or `pg` as the argument to the `-source` flag. They all specify
PostgreSQL as the source database.

Reply:
Done.